source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/UtiliOS.mm

```diff
@@ -303,7 +303,7 @@ void applyBackgroundImageConstraints(const BackgroundImage *backgroundImagePrope
                 isDeficientInHeight = YES;
             }
 
-            if (isDeficientInWidth and isDeficientInWidth) {
+            if (isDeficientInWidth and isDeficientInHeight) {
```

Comment:
isn't this change in other PR too?

Reply:
yes, it was added by cherry-pick; shouldn't matter though as the conflict can be resolved. 

---
In reply to: [387951953](https://github.com/microsoft/AdaptiveCards/pull/3818#discussion_r387951953) [](ancestors = 387951953)